src/main/java/gtPlusPlus/core/material/MISC_MATERIALS.java

```diff
@@ -15,11 +15,82 @@
 	 */
 
 	public static void run() {
+		MaterialUtils.generateSpecialDustAndAssignToAMaterial(POTASSIUM_NITRATE, false);
+		MaterialUtils.generateSpecialDustAndAssignToAMaterial(SODIUM_NITRATE, false);
 		MaterialUtils.generateSpecialDustAndAssignToAMaterial(STRONTIUM_OXIDE, false);
 		MaterialUtils.generateSpecialDustAndAssignToAMaterial(STRONTIUM_HYDROXIDE, false);
 		WATER.registerComponentForMaterial(FluidUtils.getWater(1000));
 	}
 
+	public static final Material POTASSIUM_NITRATE = new Material(
```

Comment:
Let's use saltpeter instead, which already exist in base gt since greg added railcraft compat.